### ISSUE TITLE
Public key for onlyoffice-documentserver-ie.x86_64.rpm is not installed

### ISSUE THREAD

**Issue by ShockwaveNN** [ONLYOFFICE/repo#2]
1. Add repo
2. `yum install onlyoffice-documentserver-ie`

Result:
During installation
```
warning: /var/cache/yum/x86_64/7/onlyoffice/packages/onlyoffice-documentserver-ie.x86_64.rpm: Header V4 RSA/SHA1 Signature, key ID cb2de8e5: NOKEY
Public key for onlyoffice-documentserver-ie.x86_64.rpm is not installed
```

Installation failed with message
```
Public key for onlyoffice-documentserver-ie.x86_64.rpm is not installed
```

CentOS Linux release 7.4.1708 (Core) (DigitalOcean Droplet)

**Comment by agolybev** [comment]
Fixed by https://github.com/ONLYOFFICE/repo/commit/94ce6bc355842712be33626285fb2a1b376f36da

**Comment by ShockwaveNN** [comment]
New error for https://github.com/ONLYOFFICE/repo/releases/download/v1.0.0.7/onlyoffice-repo.noarch.rpm
```
error: /etc/pki/rpm-gpg/RPM-GPG-KEY-ONLYOFFICE: key 1 not an armored public key.
warning: %post(onlyoffice-repo-1.0.0-7.noarch) scriptlet failed, exit status 1
Non-fatal POSTIN scriptlet failure in rpm package onlyoffice-repo-1.0.0-7.noarch
```
Result Public key for onlyoffice-documentserver-ie.x86_64.rpm is not installed

**Comment by ShockwaveNN** [comment]
Confirm fixed in v1.0.0.9
